Gemfile

```diff
@@ -54,6 +54,9 @@ group :development do
   gem 'capistrano-rails'
   gem 'capistrano-rbenv', '~> 2.0'
   gem 'capistrano-sidekiq'
+  gem 'rbnacl', '>= 3.2', '< 5.0'
+  gem 'rbnacl-libsodium'
+  gem 'bcrypt_pbkdf', '~> 1.0'
```

Comment:
What do the "rbnacl" gems provide? To add ed25519 support in FISbot, here is what was needed:
```
diff --git a/Gemfile b/Gemfile                                                                                                                                  │
index d22dbb4..b95ad6f 100644                                                                                                                                   │
--- a/Gemfile                                                                                                                                                   │
+++ b/Gemfile                                                                                                                                                   │
@@ -4,13 +4,11 @@ gem 'activeadmin'                                                                                                                             │
 gem 'activerecord-import'                                                                                                                                      │
 gem 'airbrake', '~> 5.6'                                                                                                                                       │
 gem 'api-pagination'                                                                                                                                           │
-gem 'bcrypt_pbkdf' # for cap deploy to Ubuntu 18                                                                                                               │
 gem 'bunny'                                                                                                                                                    │
 gem 'byebug'                                                                                                                                                   │
 gem 'clearance'                                                                                                                                                │
 gem 'daemons'                                                                                                                                                  │
 gem 'dotenv-rails'                                                                                                                                             │
-gem 'ed25519' # for cap deploy to Ubuntu 18                                                                                                                    │
 gem 'execjs'                                                                                                                                                   │
 gem 'jbuilder'                                                                                                                                                 │
 gem 'multi_daemons'
```